Football/src/navigation/StackNavigator/types.ts

```diff
@@ -0,0 +1,5 @@
+import { AuthTokenType } from '../../actions/types';
+
+export type rootState = {
```

Comment:
it's not the correct type and why type rootState locate in StackNavigator/types.ts ?